adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -653,6 +653,15 @@ private Bundle getBrokerOptions(final AuthenticationRequest request) {
         if (request.getPrompt() != null) {
             brokerOptions.putString(AuthenticationConstants.Broker.ACCOUNT_PROMPT, request.getPrompt().name());
         }
+
+        if (!StringExtensions.isNullOrBlank(request.getClaimsChallenge())) {
```

Comment:
These 2 checks do exactly the same thing, why not merge them into a single if block?